include/z64cutscene_commands.h

```diff
@@ -11,445 +11,273 @@
 #endif
 
 /**
- * ARGS
- *   s32 totalEntries (e), s32 endFrame (n)
- * FORMAT
- *   eeeeeeee nnnnnnnn
- *   size = 0x8
+ * Marks the beginning of a cutscene.
```

Comment:
```suggestion
 * Marks the beginning of a cutscene script.
```
just in case that's unclear to anyone?
(and `CS_END`)